contracts/contracts/rem.swap/src/rem.swap.cpp

```diff
@@ -50,26 +50,28 @@ namespace eosio {
             s.status         = static_cast<int8_t>(swap_status::INITIALIZED);
             if (is_producer) s.provided_approvals.push_back(rampayer);
```

Comment:
make it only possible for top25 bps to create records.

Reply:
@roman-tik, done.